### PR TITLE
New article: SEEFOOD 

### PR DESCRIPTION
**Pull request by vinayakj02**
Recreating the SEE-FOOD app from Silicon Valley using Transfer Learning in PyTorch

### CODE REVIEW

File: _posts/2022-04-12-SEEFOOD.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ - CNNs
+categories:
+ - CompSoc 
+github_username: 'vinayakj02'
+---
+
+
+
+![phone-gif](/blog/assets/img/SEEFOOD/gif1.gif) <br>
+SEEFOOD is an application which basically predicts if an image is an image of a hot-dog or not. This is based on the model used in the [HBO's Silicon Valley ](https://www.youtube.com/watch?v=vIci3C4JkL0). <br>
```
Comment (pranavdv) [inline _posts/2022-04-12-SEEFOOD.md:23]: Mention here also that the rest of the article is a tutorial on building the model. Also mention here what you expect the reader to know already (like CNNs, PyTorch, transfer learning etc).

File: _posts/2022-04-12-SEEFOOD.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+![S-E-E FOOD](/blog/assets/img/SEEFOOD/gif2.gif) <br>
+
+### How to start ? 
+First, we need to get a dataset for training the model. For this, there is a [dataset on kaggle ](https://www.kaggle.com/dansbecker/hot-dog-not-hot-dog) which perfectly matches our needs. So now that we have a dataset, we need to build and train the actual model.  
+ 
+### Let's start coding. 
+The notebook is [here](https://github.com/vinayakj02/SEEFOOD-classifier/blob/master/seefood_notebook.ipynb) if you want to follow through as you're reading the article. 
+
+First, off we import all the required libraries.
+![image](/blog/assets/img/SEEFOOD/image1.png)
```
Comment (pranavdv) [inline _posts/2022-04-12-SEEFOOD.md:33]: Don't add screenshots of the code. Write it directly in backticks. Same for all the screenshots following this as well. Cell outputs that have images can be added directly.

**Review by pranavdv — CHANGES_REQUESTED** [review]
Content is good overall. Some changes in the format are required.

File: _posts/2022-04-12-SEEFOOD.md (hunk 1 above)
Comment (vinayakj02) [reply _posts/2022-04-12-SEEFOOD.md]: Done

File: _posts/2022-04-12-SEEFOOD.md (hunk 2 above)
Comment (vinayakj02) [reply _posts/2022-04-12-SEEFOOD.md]: Done

**Review by pranavdv — APPROVED** [review]
nit: center align all images and gifs.
LGTM, nice work.

**Pull request merged**